program/rust/src/accounts/price.rs

```diff
@@ -98,20 +98,36 @@ impl PythAccount for PriceAccount {
 /// Message format for sending data to other chains via the accumulator program
 /// When serialized, each message starts with a unique 1-byte discriminator, followed by the
 /// serialized struct data in the definition(s) below.
+///
+/// Messages are forward-compatible. You may add new fields to messages after all previously
+/// defined fields. All code for parsing messages must ignore any extraneous bytes at the end of
+/// the message (which could be fields that the code does not yet understand).
 #[repr(C)]
-#[derive(Copy, Clone)]
+#[derive(Debug, Copy, Clone, PartialEq)]
 pub struct PriceFeedMessage {
-    pub id:           [u8; 32],
-    pub price:        i64,
-    pub conf:         u64,
-    pub exponent:     i32,
-    pub publish_time: i64,
-    pub ema_price:    i64,
-    pub ema_conf:     u64,
+    pub id:                [u8; 32],
+    pub price:             i64,
+    pub conf:              u64,
+    pub exponent:          i32,
+    /// The timestamp of this price update
+    pub publish_time:      i64,
+    /// The timestamp of the previous price update. This field is intended to allow users to
+    /// identify the single unique price update immediately after any moment in time:
+    /// for any time t, the unique update is the one such that prev_publish_time <= t < publish_time.
```

Comment:
`prev_publish_time < t <= publish_time` since price and conf will then be the aggregate at `publish_time` ?